PPS Shiny app/server.R

```diff
@@ -47,17 +48,19 @@ server <- function(input, output, session) {
     showModal(modalDialog(
```

Comment:
can you add the reference inlclusing the zenode doi number in the app? not really sure what place it fits best, maybe at the top of the disclaimer?

Reply:
![image](https://user-images.githubusercontent.com/60255083/94404058-53aed680-01a9-11eb-9597-d2155fb7f96b.png)

Agree with you! How about this? @Rensvandeschoot

Reply:
perferct!